ops/dev/main.tf

```diff
@@ -31,17 +25,23 @@ module "simple_report_api" {
   resource_group_location = data.azurerm_resource_group.rg.location
   resource_group_name     = data.azurerm_resource_group.rg.name
 
-  docker_image_uri = "DOCKER|simplereportacr.azurecr.io/api/simple-report-api-build:c2514f6" # hardcoding this until automated deploy of images are in place
+  docker_image_uri = "DOCKER|simplereportacr.azurecr.io/api/simple-report-api-build:7022210" # hardcoding this until automated deploy of images are in place
+  key_vault_id     = data.azurerm_key_vault.sr_global.id
+  tenant_id        = data.azurerm_client_config.current.tenant_id
 
   app_settings = {
     "DOCKER_REGISTRY_SERVER_PASSWORD"                = data.terraform_remote_state.global.outputs.acr_simeplereport_admin_password
     "DOCKER_REGISTRY_SERVER_URL"                     = "https://${data.terraform_remote_state.global.outputs.acr_simeplereport_name}.azurecr.io"
     "DOCKER_REGISTRY_SERVER_USERNAME"                = data.terraform_remote_state.global.outputs.acr_simeplereport_name
     "SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA" = "public"
     "WEBSITES_PORT"                                  = "8080"
-    SPRING_PROFILES_ACTIVE                           = "azure-dev"
+    SPRING_PROFILES_ACTIVE                           = "azure-dev,no-security"
```

Comment:
until okta auth works

Reply:
What specifically is blocking Okta from working?